src/main/java/edu/kis/vh/nursery/defaultCountingOutRhymer.java

```diff
@@ -2,33 +2,39 @@
 
 public class defaultCountingOutRhymer {
 
-    private int[] NUMBERS = new int[12];
+    private static final int NUMBERS_SIZE = 12;
+    private static final int TOTAL_BASE_VALUE = -1;
 
-    public int total = -1;
+    private final int[] numbers = new int[NUMBERS_SIZE];
+
+    private int total = TOTAL_BASE_VALUE;
 
     public void countIn(int in) {
         if (!isFull())
-            NUMBERS[++total] = in;
+            numbers[++total] = in;
     }
 
-        public boolean callCheck() {
-            return total == -1;
-        }
-        
-            public boolean isFull() {
-                return total == 11;
-            }
-        
-                protected int peekaboo() {
-                    if (callCheck())
-                        return -1;
-                    return NUMBERS[total];
-                }
-            
-                    public int countOut() {
-                        if (callCheck())
-                            return -1;
-                        return NUMBERS[total--];
-                    }
+    public boolean callCheck() {
+        return total == TOTAL_BASE_VALUE;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).
This constant should be different (separate) than the one in peekaboo and countOut methods.